### ISSUE TITLE
[BUG] Failed to run example code in README

### ISSUE THREAD

**Issue by xiazhicheng** [mars-project/mars#3108]
Code in README run failed

```python
import numpy as np
import mars.remote as mr

def calc_chunk(n, i):
    rs = np.random.RandomState(i)
    a = rs.uniform(-1, 1, size=(n, 2))
    d = np.linalg.norm(a, axis=1)
    return (d < 1).sum()

def calc_pi(fs, N):
    return sum(fs) * 4 / N

N = 200_000_000
n = 10_000_000

fs = [mr.spawn(calc_chunk, args=(n, i))
      for i in range(N // n)]
pi = mr.spawn(calc_pi, args=(fs, N))
print(pi.execute().fetch())
```

While the code below returned correct results:

```python
import numpy as np
import mars.remote as mr


def calc_chunk(n, i):
    rs = np.random.RandomState(i)
    a = rs.uniform(-1, 1, size=(n, 2))
    d = np.linalg.norm(a, axis=1)
    return (d < 1).sum()

def calc_pi(fs, N):
    return sum(fs) * 4 / N

N = 200_000_000
n = 10_000_000

def main():
    fs = [mr.spawn(calc_chunk, args=(n, i))
        for i in range(N // n)]
    pi = mr.spawn(calc_pi, args=(fs, N))
    print(pi.execute().fetch())


if __name__=="__main__":
    main()
```

As multiprocessing will try importing `__main__` module and code is reexecuted again.

**Comment by hekaisheng** [comment]
Could you please add a title and translate issue into English, thanks in advance.

**Comment by wjsi** [comment]
We added support for creating local clusters without need to add `__name__ == "__main__"` protection in #3110 and it is expected to be released with v0.9.0. Thanks for the report.
